api/types/role.go

```diff
@@ -159,80 +170,80 @@ const (
 )
 
 // GetVersion returns resource version
-func (r *RoleV4) GetVersion() string {
+func (r *RoleV5) GetVersion() string {
```

Comment:
V5 "replaces" V4. The protobuf scheme is backwards compatible with V4 so no explicit conversion. Newer fields from V5 will be discarded if the client expects V4 and V4 -> V5 works since the new V5 fields get zero values.